pkg/crc/machine/machine.go

```diff
@@ -284,20 +284,6 @@ func (client *client) Start(startConfig StartConfig) (StartResult, error) {
 		return startError(startConfig.Name, "Error getting the IP", err)
 	}
 
-	var hostIP string
-	determineHostIP := func() error {
-		hostIP, err = network.DetermineHostIP(instanceIP)
```

Comment:
oh! this err was written in place of this err declaration:

```
exists, err := client.Exists(startConfig.Name)
```

lint error appear now because we never touch this variable.